.pipelines/cs-build-steps.yaml

```diff
@@ -12,7 +12,7 @@ steps:
     performMultiLevelLookup: true
     useGlobalJson: true
 
-- task: NuGetAuthenticate@0
+- task: NuGetAuthenticate@1
```

Comment:
I noticed that the @0 version of this task is deprecated: https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/nuget-authenticate-v0?view=azure-pipelines